cs-dropbox/src/main/java/io/cloudslang/content/dropbox/actions/folders/CreateFolder.java

```diff
@@ -0,0 +1,188 @@
+     */
+    @Action(name = "Create Folder",
+            outputs = {
+                    @Output(RETURN_CODE),
+                    @Output(RETURN_RESULT),
+                    @Output(EXCEPTION)
+            },
+            responses = {
+                    @Response(text = SUCCESS, field = RETURN_CODE, value = ReturnCodes.SUCCESS,
+                            matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.RESOLVED),
+                    @Response(text = FAILURE, field = RETURN_CODE, value = ReturnCodes.FAILURE,
+                            matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.ERROR, isOnFail = true)
+            })
+    public Map<String, String> execute(@Param(value = ENDPOINT, required = true) String endpoint,
+                                       @Param(value = ACCESS_TOKEN, required = true, encrypted = true) String accessToken,
+                                       @Param(value = API_VERSION) String apiVersion,
+                                       @Param(value = PROXY_HOST) String proxyHost,
+                                       @Param(value = PROXY_PORT) String proxyPort,
+                                       @Param(value = PROXY_USERNAME) String proxyUsername,
+                                       @Param(value = PROXY_PASSWORD, encrypted = true) String proxyPassword,
+                                       @Param(value = TRUST_ALL_ROOTS) String trustAllRoots,
+                                       @Param(value = X509_HOSTNAME_VERIFIER) String x509HostnameVerifier,
+                                       @Param(value = TRUST_KEYSTORE) String trustKeystore,
+                                       @Param(value = TRUST_PASSWORD, encrypted = true) String trustPassword,
+                                       @Param(value = KEYSTORE) String keystore,
+                                       @Param(value = KEYSTORE_PASSWORD, encrypted = true) String keystorePassword,
+                                       @Param(value = CONNECT_TIMEOUT) String connectTimeout,
+                                       @Param(value = SOCKET_TIMEOUT) String socketTimeout,
+                                       @Param(value = USE_COOKIES) String useCookies,
+                                       @Param(value = KEEP_ALIVE) String keepAlive,
+                                       @Param(value = AUTO_RENAME) String autorename,
+                                       @Param(value = PATH, required = true) String path) {
+        try {
+            final HttpClientInputs httpClientInputs = getHttpClientInputs(proxyHost, proxyPort, proxyUsername, proxyPassword,
+                    trustAllRoots, x509HostnameVerifier, trustKeystore, trustPassword, keystore, keystorePassword, connectTimeout,
+                    socketTimeout, useCookies, keepAlive, METHOD_NAME);
+
+            final CommonInputs commonInputs = new CommonInputs.Builder()
+                    .withAction(CREATE_FOLDER)
```

Comment:
@Tirla-Alin Yes... version 1 is deprecated, version 2 is on use... but what if another API version will appear... wouldn't be great to have control on them?